blog/_posts/2018-11-09-dvteam-farewell

```diff
@@ -1,51 +1,45 @@
 ---
```

Comment:
What is still missing here is the extension in the filename. The file should be called `2018-11-09-dvteam-farewell.md` instead of `2018-11-09-dvteam-farewell`, that way it gets recognised by Jekyll / the system as a markdown file, and the information in it gets displayed accordingly. Could you still change that @artnerdnet? Thanks! 🙌

Reply:
done